CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.33.2
+* Filters should now work for all columns in the Results Table component, even for the one paused in the greyVest example i.e. `Title`.
```

Comment:
This is should be more clear about what was fixed, e.g.

```
ResultsTable: Inline filtering on paused nodes will now be unpaused instead of silently failing
GreyVest: Added a paused node example
```